### PR TITLE
Fix YouTube URL field in Video Player module to allow for hyphens in the video ID

### PR DESCRIPTION
**Pull request by Scotchester**
Short description explaining the high-level reason for the pull request: YouTube is dumb and allows hyphens in their video IDs.

## Changes

- Now correctly allows for hyphens in the video ID of a Video Player's `video_url` field.

## Testing

- Add a video player module to a Learn page and test that you can give it the following URL and it won't throw a validation error: https://www.youtube.com/embed/MD5iOesN-2o

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
* [x] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by sebworks** [comment]
lol, I never knew. What about dashes?

**Comment by Scotchester** [comment]
> What about dashes?

What about them?

**Comment by Scotchester** [comment]
I guess this needs a migration. Coming soon.

**Comment by sebworks** [comment]
Do they allow dashes in videos ids?

**Comment by Scotchester** [comment]
If they allow both hyphens AND dashes, I'll eat my hat.

File: cfgov/v1/atomic_elements/organisms.py [hunk 1]
```diff
@@ -506,7 +506,7 @@ class VideoPlayer(blocks.StructBlock):
         label='YouTube Embed URL',
         default='https://www.youtube.com/embed/',
         required=True,
-        regex=r'^https:\/\/www\.youtube\.com\/embed\/[a-zA-Z0-9]+([/]?)$',
+        regex=r'^https:\/\/www\.youtube\.com\/embed\/[a-zA-Z0-9\-]+([/]?)$',
```
Comment (contolini) [inline cfgov/v1/atomic_elements/organisms.py]: Underscores are valid as well (https://www.youtube.com/watch?v=_-G520ACFUo).

Comment (Scotchester) [inline cfgov/v1/atomic_elements/organisms.py]: Thanks, Chris. Steven also pointed that out in an off-thread chat. We decided to just make it significantly more forgiving for the time being, with the intention of refactoring the rendering macro in the near future to accept any valid YouTube URL, or even just the ID.

**Comment by Scotchester** [comment]
@sebworks @contolini Commit pushed to make it more forgiving, and add the needed migration.

**Pull request merged**